charts/bpa/templates/acapy_deployment.yaml

```diff
@@ -31,19 +31,21 @@ spec:
           imagePullPolicy: {{ .Values.acapy.image.pullPolicy }}
           args: [
            "-c",
-           "sleep 15; \
-           aca-py start \  
+           "curl -d '{\"seed\":\"$(WALLET_SEED)\", \"role\":\"TRUST_ANCHOR\", \"alias\":\"{{ include "bpa.fullname" . }}\"}' -X POST {{ .Values.bpa.config.ledger.browser }}/register; \
+           sleep 15; \
+           aca-py start \     
+           --auto-provision \
            --arg-file acapy-static-args.yml \   
            --inbound-transport http '0.0.0.0' {{ .Values.acapy.service.httpPort }} \ 
            --webhook-url http://{{ include "bpa.fullname" . }}:{{ .Values.bpa.service.port }}/log \
            --genesis-url '{{ .Values.bpa.config.ledger.browser }}/genesis' \
-           --endpoint https://{{ (index .Values.acapy.ingress.hosts 0).host }} \
+           --endpoint http://{{ .Release.Name }}-{{ (index .Values.acapy.ingress.hosts 0).host }} \
```

Comment:
Thats a great idea. I also wanted to refine the usage of ingress host values.
Maybe we could even refactor the complete ingress definitions ( acapy and bpa):
- define a value "ingress suffix" which is basically the domain "example.com".
- default behaviour: enable ingresses;  generate ingress definitions based on this suffix and include release name 
- generated ingress could be optionally overwritten ( to allow  complex "manually defined"  ingress defintions in case someone needs that)